crates/bevy_ui/src/render/mod.rs

```diff
@@ -222,30 +225,46 @@ const UI_CAMERA_FAR: f32 = 1000.0;
 // TODO: Evaluate if we still need this.
 const UI_CAMERA_TRANSFORM_OFFSET: f32 = -0.1;
 
-#[derive(Component)]
-pub struct DefaultCameraView(pub Entity);
+/// The UI camera used by this Camera's viewport.
+///
+/// This component is inserted into the render world in the
+/// [`extract_default_ui_camera_view`] system.
+///
+/// The component is attached to the "actual" viewport's camera.
```

Comment:
Ugh, this is making me second guess removing explicitly initialized UI cameras.  Seems like the right design if we're not changing that though.